src/utils/logging_config.py

```diff
@@ -9,23 +9,23 @@
 
 logger = logging.getLogger(__name__)
 
-def configure_logging(log_dir, config) -> None:
+def configure_logging(log_folder, config) -> None:
```

Comment:
I recommend using type definitions in function definitions. It means the code is more self-documenting and can really aid development
```suggestion
def configure_logging(log_folder: Path, config: Dict[str, Any]) -> None:
```
This changed would be something that should be propegated to the rest of the repository, if it is a standard you want to adopt.

Reply:
cool, raised issue: https://github.com/NHSDigital/RAP_example_pipeline_python/issues/9